### PR TITLE
Add metadata to test templates

### PR DESCRIPTION
**Pull request by dlazin**

### CODE REVIEW

File: tests/xx-epub-template/OPS/package.opf [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+        TODO: A longer description of the test for the generated test report.
+    </dc:description>
+    <dc:coverage>TODO: Report category</dc:coverage>
+    <meta property="dcterms:isReferencedBy">TODO: https://www.w3.org/TR/epub-33/#spec-anchor</meta>
+    <meta property="dcterms:isReferencedBy">TODO: https://www.w3.org/TR/epub-rs-33/#spec-anchor</meta>
+
+    <dc:creator>W3C EPUB 3 Working Group</dc:creator>
     <dc:language>en</dc:language>
+    <dc:date>2021-01-01</dc:date>
     <meta property="dcterms:modified">2021-01-01T00:00:00Z</meta>
```
Comment (dlazin) [inline tests/xx-epub-template/OPS/package.opf:16]: I have this organized in two sections — stuff up top that needs to be modified for every test, in the order documented at https://github.com/w3c/epub-tests/tree/instructions#metadata, followed by these properties at the bottom that (generally) don't need to be changed.

How do we feel about a) that and b) the general order? Open to changing it.

File: tests/xx-epub-template/OPS/package.opf [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+    <dc:identifier id="pub-id">TODO: spec-anchor</dc:identifier>
+    <dc:title>TODO: Concise Test Title</dc:title>
+    <dc:description>
+        TODO: A longer description of the test for the generated test report.
+    </dc:description>
+    <dc:coverage>TODO: Report category</dc:coverage>
+    <meta property="dcterms:isReferencedBy">TODO: https://www.w3.org/TR/epub-33/#spec-anchor</meta>
+    <meta property="dcterms:isReferencedBy">TODO: https://www.w3.org/TR/epub-rs-33/#spec-anchor</meta>
+
+    <dc:creator>W3C EPUB 3 Working Group</dc:creator>
```
Comment (dlazin) [inline tests/xx-epub-template/OPS/package.opf:13]: How do you feel about this creator vs putting the respective test author's name in the file?

Comment (iherman) [reply tests/xx-epub-template/OPS/package.opf:13]: I am, mildly, in favour of using the author's name.

File: tests/xx-epub-template/OPS/package.opf (hunk 1 above)
Comment (iherman) [reply tests/xx-epub-template/OPS/package.opf:16]: This looks o.k. to me. My script is not order dependent, b.t.w., so whatever looks o.k. works.

**Review by iherman — APPROVED** [review]

File: tests/xx-epub-template/OPS/package.opf (hunk 1 above)
Comment (dlazin) [reply tests/xx-epub-template/OPS/package.opf:15]: I decided to go with strict alphabetization. It's the only scheme that isn't subjective and anyone can understand (even if it is not _optimal_ to some eyes).

File: tests/xx-epub-template/OPS/package.opf (hunk 2 above)
Comment (dlazin) [reply tests/xx-epub-template/OPS/package.opf]: Done.

**Pull request merged**